tracker/classify.go

```diff
@@ -0,0 +1,42 @@
+// Package tracker implements an object tracker as a Viam vision service
+// This file contains methods that are useful for classifying the detections
+package tracker
+
+import (
+	"context"
+	"image"
+
+	"go.viam.com/rdk/rimage"
+	"go.viam.com/rdk/services/vision"
+	objdet "go.viam.com/rdk/vision/objectdetection"
+)
+
+// method will take a slice of tracks and return a slice of tracks.
+// The only difference is that the track will now include the detection classification
+func classifyTracks(ctx context.Context, tracks []*track, img image.Image, classifier vision.Service) []*track {
+	
+	for _, tr := range tracks {
+		cropped := cropImageFromDet(img, tr.Det)
+		out, err := classifier.Classifications(ctx, cropped, 1, nil)
```

Comment:
Isn't the classifier optional? classifier could be nil. This would panic. You should check first to see if the classifier is set, either in this function, or before we reach this function.